File: sukh_1684/problems/remove_duplicates_from_sorted_array/solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length==0){
            return 0;
        }
        int pointer=1;
        int current=nums[0];
        for(int i=1;i<nums.length;i++){
            if(nums[i]!=current){
               current = nums[i];
               swap(nums,i,pointer);
                
                pointer++;
            }
        }
        return pointer;
        
    }
    private void swap(int[] nums, int left, int right){
        int temp = nums[left];
        nums[left] = nums[right];
        nums[right]=temp;
    }
}

File: sukh_1684/problems/remove_duplicates_from_sorted_array/solution.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        current = 0
        length = len(nums)
        for i in range(1,length):
            if(nums[i] > nums[current]):
                current += 1
                nums[i], nums[current] = nums[current], nums[i]
        
                
            
            
        
        return current+1